### PR TITLE
Proofreading IMC's lines - Part 6

### PR DESCRIPTION
**Pull request by Sunaries**
Proofread:

-https://github.com/Treeofsavior/EnglishTranslation/commit/a7bb10cca4ddbcc0f3ef9ad28fe568d4d04bfa65
-https://github.com/Treeofsavior/EnglishTranslation/commit/5280bbd8dcbce6e3d82027c6e36add3b2f89018f
-https://github.com/Treeofsavior/EnglishTranslation/commit/a59c28aa32b62692958718629b1ebdc49131c1f5
- Fixed many typos, ungrammatical sentences, inconsistencies, incorrect ellipses etc
- Reworded sentences and replaced {nl} tags etc for better quality English

### CODE REVIEW

File: ETC.tsv [hunk 1]
```diff
@@ -18103,74 +18103,74 @@ ETC_20151224_018104
...
+ETC_20151224_018142	in towns and cities.
 ETC_20151224_018143	Buy-in Shop
-ETC_20151224_018144	You can open the shop through the character setting tap by pressing {img F1 40 40}.
-ETC_20151224_018145	{nl} {nl}After set the shop name and set the price of the quantity of each by searching the purchased items,  Enroll the purchsed items to press the triangle button
-ETC_20151224_018146	If the desired items are completed for enrollment, you can open the Buy-in shop using the button below.
-ETC_20151224_018147	{nl} {nl}The other player are able to sell the desired items through the opened the Buy-in shop.
-ETC_20151224_018148	The player who opened the Buy-in shop can't move, but even if logout the character, the Buy-in shop is still running.
+ETC_20151224_018144	You can open the shop through the Character Settings tab by pressing {img F1 40 40}.
+ETC_20151224_018145	{nl} {nl}After setting the shop name and the price per unit of the items you want, press the triangular button to confirm.
+ETC_20151224_018146	If the wanted items are completed for enrollment, you can open the Buy-in Shop using the button below.
```
Comment (Sunaries) [inline ETC.tsv:18143]: Note for myself: change "enrollment".

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**